libs/upd/components/src/lib/data-table/data-table.component.css

```diff
@@ -1,3 +1,12 @@
 .bg-green {
   background-color: #90ee90 !important;
 }
+/* .bg-red {
+  background-color: #f5bcbc  !important;
+}
+.bg-orange {
+  background-color: #f8e6d2  !important;
+} */
+td.validation-kpi-not-met + td.bg-green {
```

Comment:
At a glance it's really hard to tell why this is set up this way. Seems very error-prone and could be hard to debug for someone that doesn't know this rule exists. In general it's probably not a good idea to embed logic into CSS rules and instead just manipulate CSS classes or styles explicitly in your code.